src/TLuaInterpreter.cpp

```diff
@@ -2477,6 +2477,28 @@ int TLuaInterpreter::closeMudlet(lua_State* L)
     return 0;
 }
 
+int TLuaInterpreter::saveProfile(lua_State* L)
+{
+    Host* pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if (!pHost) {
+        lua_pushstring(L, QLatin1String("saveProfile: NULL Host pointer - something is wrong!").data());
+        lua_error(L);
+        return 2;
+    }
+
+    auto result = pHost->saveProfile();
+
+    if (result.first == true) {
+        lua_pushboolean(L, true);
+        return 1;
+    } else {
+        lua_pushnil(L);
+        QString msg = QLatin1String("saveProfile: ") + result.second;
```

Comment:
This was my attempt at following [this guideline](https://github.com/Mudlet/Mudlet/blob/development/CONTRIBUTING.md#use-qlatin1string-over-qstringliteral-if-possible) while keeping in mind that [Effective Qt](https://meetingcpp.com/tl_files/mcpp/2015/talks/Marc-Mutz-MC++15-Effective-Qt.pdf) says:

> Prefer
> – QString s = ... + ... + ... + ...; // QStringBuilder